src/Orchard.Web/Modules/Orchard.ContentPicker/Views/RecentContentTab.cshtml

```diff
@@ -1,5 +1,7 @@
 ﻿@using Orchard.Core.Contents.ViewModels;
 @{
+    Script.Include("RecentContentTab.js");

```

Comment:
This file is in the resource manifest so should this be `Script.Require("RecentContentTab");`?